src/main/java/wooteco/subway/service/member/MemberService.java

```diff
@@ -18,34 +21,42 @@ public MemberService(MemberRepository memberRepository, JwtTokenProvider jwtToke
     }
 
     public Member createMember(Member member) {
-        return memberRepository.save(member);
-    }
-
-    public void updateMember(Long id, UpdateMemberRequest param) {
-        Member member = memberRepository.findById(id).orElseThrow(RuntimeException::new);
-        member.update(param.getName(), param.getPassword());
-        memberRepository.save(member);
-    }
-
-    public void deleteMember(Long id) {
-        memberRepository.deleteById(id);
-    }
-
-    public String createToken(LoginRequest param) {
-        Member member = memberRepository.findByEmail(param.getEmail()).orElseThrow(RuntimeException::new);
-        if (!member.checkPassword(param.getPassword())) {
-            throw new RuntimeException("잘못된 패스워드");
+        String email = member.getEmail();
+        if (memberRepository.findByEmail(email).isPresent()) {
```

Comment:
```suggestion
        if (memberRepository.existsByEmail(email)) {
```
MemberRepository에 boolean 타입을 반환하는 existsByEmail 메서드를 추가해보아요